jobs/ci-kubernetes-e2e-cos-docker-validation-serial.env

```diff
@@ -1,5 +1,5 @@
-# Use GCI builtin k8s version.
-JENKINS_USE_GCI_VERSION=y
+# Use fixed k8s version.
+JENKINS_PUBLISHED_VERSION=v1.6.1
```

Comment:
Although this is probably better as `ci/latest-1.6` so that we can patch test changes in if necessary.